aip/general/0203.md

```diff
@@ -42,6 +42,14 @@ backwards-compatility. Nontheless, this annotation **must not** be omitted.
 only, and does not itself add any validation. The purpose is to consistently
 document this behavior for clients.
 
+### Annotating Name
+
+The `name` field of a resource **must not** be annotated.
+
+Since the resource name (see AIP-122) is used in a variety of contexts in which some are
+`OUTPUT_ONLY`, `REQUIRED`, and `IMMUTABLE`, clients **should** have special
+handling for that field based on the request.
```

Comment:
> clients **should** have special handling for that field based on the request

IDK how actionable/enforceable this is but I understand we want to call it out as a special case that needs to be recognized...perhaps don't use a requirement word? idk